AK/NumberFormat.h

```diff
@@ -34,6 +34,7 @@ namespace AK {
 static String number_string_with_one_decimal(float number, const char* suffix)
 {
     float decimals = number - (int)number;
+    // FIXME: Remove this hackery once printf() supports floats.
```

Comment:
But `printf()` does support floats now, so let's just use that instead? :)

Reply:
woah this wasn't a thing last time I looked, very nice :)